src/services/organization-extension.js

```diff
@@ -21,8 +22,35 @@ module.exports = class orgExtensionsHelper {
 	static async createConfig(bodyData, organization_id) {
 		try {
 			bodyData.organization_id = organization_id
-			const { resource_type, review_stages, review_type } = bodyData
+			const { resource_type, review_stages, review_type, data_managers } = bodyData
+			// check if body have data_managers
+			if (data_managers && data_managers.length > 0) {
```

Comment:
@adithyadinesh0412 There should be a single query of create in this function. cc - @priyanka-TL